tests/python/test_fake.py

```diff
@@ -4,6 +4,26 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+import pytest
+import torch
 
-def test_foo():
-    assert True
+from torchdistx.fake import fake_mode, meta_like
+
+
+def test_meta_like_works_as_expected():
+    with fake_mode():
+        a = torch.ones([10])
+
+    b = meta_like(a)
+
+    assert b.device.type == "meta"
+    assert b.dtype == a.dtype
+    assert b.size() == a.size()
+    assert b.stride() == a.stride()
```

Comment:
Shall we test more complex cases such as non-contiguous or sparse tensors?

Reply:
There is no such thing as a sparse meta tensor (equivalently sparse fake tensor). A meta/fake tensor always represents a dense tensor. COO and CSR tensors in PyTorch are build as compositions of dense tensors but their meta backend coverage for sparse operators is pretty weak for now.

Reply:
I see, thanks for explaining!